raiden/network/channelgraph.py

```diff
@@ -90,7 +85,23 @@ def channel_to_routestate(channel, node_address):
     return state
 
 
+#  To sort multiple-keyed list we use redefined __lt__ method,
+# which is then used by sorted()
+class RouteOrderedItem:
+    def __init__(self, distance_to_target, node_state, route_state):
+        assert isinstance(route_state, RouteState)
+        self.distance_to_target = distance_to_target  # key1
+        self.node_state = node_state                  # key2
+        self.route_state = route_state                # value
+
+    def __lt__(self, other):
```

Comment:
it's missing the other operators and this should have a total order, use `functools.total_ordering` and write tests to check the ordering constraints.